Jaisai.md

```diff
@@ -0,0 +1,26 @@
+Jaswanth Kumar Jonnalagadda, Maryville,MO/CST
+
+I am a passionate and committed Software developer with one year of work experience in Tata Consultancy services, INDIA (Multi National IT Service Company).
+
+• Posses good knowledge over MYSQL and MongoDB usage.
```

Comment:
There is a better way to create list you can see [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists).